modules/client-configuration/pages/clients-rh-rhui.adoc

```diff
@@ -335,7 +335,7 @@ The channels you need for this procedure are:
 | OS Version | Base Product          | Base Channel
 | {redhat} 6 | RHEL6 Base x86_64 | rhel6-pool-x86_64
 | {redhat} 7 | RHEL7 Base x86_64 | rhel7-pool-x86_64
-| {redhat} 8 | RHEL or SLES ES or CentOS 8 Base | rhel8-pool-x86_64
+| {redhat} 8 | RHEL or SLES ES | rhel8-pool-x86_64
```

Comment:
The channel / product name does not change just because CentOS 8 is now unsupported

Reply:
reverted